marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/writer/HandleAbortedWriterTest.java

```diff
@@ -31,8 +31,14 @@ void dataSourceWriterIsAborted() {
         assertEquals(2, getFruitCount(), "Just verifying the two fruits were written");
 
         dataSourceWriter.abort(null);
-        assertEquals("canceled", getJobDocumentStatus(), "If the DataSourceWriter is aborted for any reason, the job " +
-            "document should have a status of 'canceled'. Note that this does not imply whether any writes failed. " +
-            "But that is a limitation of the JobStatus class in DHF.");
+        if(canUpdateJobDoc()){
+            assertEquals("canceled", getJobDocumentStatus(), "If the DataSourceWriter is aborted for any reason, the job " +
+                "document should have a status of 'canceled'. Note that this does not imply whether any writes failed. " +
+                "But that is a limitation of the JobStatus class in DHF.");
+        }
+        else{
+            assertEquals("started", getJobDocumentStatus(), "In case of 9.0-x server, status will remain 'started'");
```

Comment:
Do we know why this is not working on 9.0-x? Is it something about amps and SJS functions? Whatever the reason is, it should be part of the assertion message.

Reply:
Actually, this can become a new protected method in AbstractSparkConnectorTest - verifyJobDocumentWasNotUpdated(). Then you can reuse it across tests.